arch/risc-v/src/esp32c3/esp32c3_rtc.c

```diff
@@ -210,18 +214,22 @@
  */
 
 #define RTC_CONFIG_DEFAULT() {\
-    .ck8m_wait = RTC_CNTL_CK8M_WAIT_DEFAULT, \
-    .xtal_wait = RTC_CNTL_XTL_BUF_WAIT_DEFAULT, \
-    .pll_wait  = RTC_CNTL_PLL_BUF_WAIT_DEFAULT, \
-    .clkctl_init = 1, \
-    .pwrctl_init = 1, \
-    .rtc_dboost_fpd = 1, \
-    .xtal_fpu = 0, \
-    .bbpll_fpu = 0, \
-    .cpu_waiti_clk_gate = 1, \
-    .cali_ocode = 0\
+  .ck8m_wait = RTC_CNTL_CK8M_WAIT_DEFAULT, \
+  .xtal_wait = RTC_CNTL_XTL_BUF_WAIT_DEFAULT, \
+  .pll_wait  = RTC_CNTL_PLL_BUF_WAIT_DEFAULT, \
+  .clkctl_init = 1, \
+  .pwrctl_init = 1, \
+  .rtc_dboost_fpd = 1, \
+  .xtal_fpu = 0, \
+  .bbpll_fpu = 0, \
+  .cpu_waiti_clk_gate = 1, \
+  .cali_ocode = 0\
 }
 
+#ifdef CONFIG_RTC_DRIVER
+#  define MAGIC_RTC_SAVE (0x11223344556677ull)
```

Comment:
pls,
add a description for this number (a justification)

Reply:
Done, thanks for the suggestion.